src/main/java/org/jabref/logic/citationstyle/CSLAdapter.java

```diff
@@ -66,7 +67,8 @@
     private void initialize(String newStyle, CitationStyleOutputFormat newFormat) throws IOException {
         if ((cslInstance == null) || !Objects.equals(newStyle, style)) {
             // lang and forceLang are set to the default values of other CSL constructors
-            cslInstance = new CSL(dataProvider, new JabRefLocaleProvider(), newStyle, "en-US", false);
+            cslInstance = new CSL(dataProvider, new JabRefLocaleProvider(),
+                  new DefaultAbbreviationProvider(), null, newStyle, "en-US", false, true);
```

Comment:
Nitpick: Does CSL also provide a builder pattern (see https://github.com/HugoMatilla/Effective-JAVA-Summary#2-use-builders-when-faced-with-many-constructors). Passing "null" and "false" and "true" is wired ^^.

Reply:
Good catch! Actually, almost all classes from citeproc-java provide a Builder except for the most important one 😂I'll add this in the next version.